Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/global.d.ts

```diff
@@ -57,7 +57,9 @@ export namespace OpenXDA {
     //Events
     interface Event { UpdatedBy: string, FileVersion: number, Description: string, SamplesPerCycle: number, SamplesPerSecond: number, TimeZoneOffset: number, Samples: number, EndTime: string, StartTime: string, Alias: string, Name: string, EventDataID: number | null, EventTypeID: number, AssetID: number, MeterID: number, FileGroupID: number, ID: number, ShortName: string }
 
-    interface EventChannel extends GemstoneXDA.Types.Channel { SourceIndices: string, MeterID: number, AssetID: number, PhaseID: number, MeasurementTypeID: number   }
+    interface EventChannel extends GemstoneXDA.Types.Channel { SourceIndices: string, MeterID: number, AssetID: number, PhaseID: number, MeasurementTypeID: number }
+    interface TrendChannel extends GemstoneXDA.Types.Channel { MeterID: number, AssetID: number, PhaseID: number, MeasurementTypeID: number, MeasurementCharacteristicID: number }
```

Comment:
Assuming you distinguish between `Trend` and `Event` Channels because of the ``MeasurementCharacteristicID` and `SourceIndices`?

Reply:
Yes, as far as I understand, trend channels don't utilize source indices

Reply:
It's more complicated than that. TrendChannels have multiple `Series` objects so they have (up to) 3 different Source indices. As a result we do not display them in the Tables. Although it may be worth doing so at some point in the future.....